src/cljs/main/broadfcui/page/workspace/notebooks/notebooks.cljs

```diff
@@ -65,6 +65,21 @@
 (defn create-inline-form-label [text]
   [:span {:style {:marginBottom "0.16667em" :fontSize "88%"}} text])
 
+(defn- leo-notebook-url [cluster]
+  (str (config/leonardo-url-root) "/notebooks/" (:googleProject cluster) "/" (:clusterName cluster)))
+
+(defn- open-notebook-in-new-tab [url]
+  (let [notebook-window (.open js/window url "_blank")]
+    (.addEventListener js/window "message"
+      (fn [e]
+        (when
```

Comment:
Tiniest of tiny nitpicks related to formatting - For `when` and `and`, they should never be on a line by themselves